src/kaleidoscope/driver/led/Gamma.h

```diff
@@ -0,0 +1,47 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::driver::led::Gamma -- Gamma correction table
```

Comment:
we probably want to figure out a slightly different name, since the assumption is that everything inside driver/led is an LED Driver class

Reply:
Agreed. This was just a quick way to lift it out.

How about, say, `kaleidoscope::util::Gamma`?

Reply:
Or... `kaleidoscope::color::GammaCorrection`? There's a couple of other things currently in `::driver::led` we could move to `::color`.

Reply:
Moved it to `::driver::color::gamma_correction` for now.